valkey.conf

```diff
@@ -1782,22 +1782,28 @@ aof-timestamp-enabled no
 #
 # In order to make a cluster work in such environments, a static
 # configuration where each node knows its public address is needed. The
-# following four options are used for this scope, and are:
+# following options are used for this scope, and are:
 #
 # * cluster-announce-ip
+# * cluster-announce-client-ipv4
+# * cluster-announce-client-ipv6
```

Comment:
How about give client one example for the ipv6 format, because in valkey, ipv6 always as [::1]:1234

Reply:
Yes, I'll add it under Examples, at your other comment.

Reply:
In CLIENT LIST, IPv6 and port is displayed as `[::1]:6379` but in redirects it's like `::1:6379` (e.g. `-MOVED 16383 ::1:6379`), so clients need to look for the last ":".